apps/managedidentity/managedidentity_test.go

```diff
@@ -715,15 +721,10 @@ func TestAzureMLAcquireTokenReturnsTokenSuccess(t *testing.T) {
 			if result.AccessToken != token {
 				t.Fatalf("wanted %q, got %q", token, result.AccessToken)
 			}
-			if testCase.miType.value() == "clientId" {
-				if query.Get("clientid") != "clientId" {
-					t.Fatalf("expected clientid to be set to clientId, got %s", query.Get("clientid"))
-				}
-			} else {
-				if query.Get("clientid") != os.Getenv("DEFAULT_IDENTITY_CLIENT_ID") {
-					t.Fatalf("expected clientid to be set to default identity client id, got %s", query.Get("clientid"))
-				}
+			if query.Get("clientid") != testCase.expectedClientID {
+				t.Fatalf("expected clientid to be set to %s, got %s", testCase.expectedClientID, query.Get("clientid"))
```

Comment:
just a suggestion for tidying and reducing the chance of typo bugs
```suggestion
			if actual := query.Get("clientid"); actual != testCase.expectedClientID {
				t.Fatalf("expected clientid to be set to %s, got %s", testCase.expectedClientID, actual)
```